tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/LockFileTestCase.java

```diff
@@ -29,6 +29,7 @@
 import org.testng.annotations.Test;
 import org.wso2.ballerinalang.compiler.util.ProjectDirConstants;
 import org.wso2.ballerinalang.programfile.ProgramFileConstants;
```

Comment:
These tests are outdated now can we clean them up.

Reply:
Removed unwanted tests